cmd/deploy/deploy.go

```diff
@@ -392,7 +385,7 @@ func (dc *Command) Run(ctx context.Context, deployOptions *Options) error {
 	dc.IoCtrl.Logger().Debugf("manifest path to store in metadata: %q", manifestPathForConfigMap)
 	data := &pipeline.CfgData{
 		Name:       deployOptions.Name,
-		Namespace:  deployOptions.Manifest.Namespace,
+		Namespace:  okteto.GetContext().Namespace,
```

Comment:
I think we should use the same context during the function. On line 352 we are using `deployOptions.Namespace`. Is there any reason for using `okteto.Context` vs `deployOptions`?

Reply:
good catch, addressed in b03660f7567bb946bdfb878350840e2623ff0542